js/piwik.js

```diff
@@ -7184,11 +7219,11 @@ if (typeof window.Piwik !== 'object') {
  *
  * @param string documentTitle
  * @param int|string siteId
- * @param string piwikUrl
+ * @param string matomoUrl
  * @param mixed customData
  */
 if (typeof piwik_log !== 'function') {
-    piwik_log = function (documentTitle, siteId, piwikUrl, customData) {
+    piwik_log = function (documentTitle, siteId, matomoUrl, customData) {
```

Comment:
Do we want there to be a matomo_log, matomo_ignore_classes, etc. globals?

Reply:
@diosmosis that's there for BC so we can't change it. I guess 0.X or 1.X versions used this `piwik_log()`

Reply:
I see 👍